creator-node/src/components/contentBlacklist/contentBlacklistComponentService.js

```diff
@@ -0,0 +1,61 @@
+const BlacklistManager = require('../../blacklistManager')
+
+const getAllContentBlacklist = async () => {
+  const { trackIdsToBlacklist, userIdsToBlacklist } = await BlacklistManager.getTrackAndUserIdsToBlacklist()
+  return { trackIds: trackIdsToBlacklist, userIds: userIdsToBlacklist }
+}
+
+const addToContentBlacklist = async ({ type, id }) => {
+  let resp
+  try {
+    // add to ContentBlacklist
+    resp = await BlacklistManager.addToDb({ id, type })
+
+    // add to redis
+    switch (resp.type) {
+      case 'USER': {
+        await BlacklistManager.add([], [resp.id])
+        break
+      }
+      case 'TRACK': {
+        await BlacklistManager.add([resp.id])
+        break
+      }
+    }
+  } catch (e) {
+    throw e
```

Comment:
no real point to doing this if u just re-throw, but doesn't really make a diff

Reply:
good point! will refactor